src/mumble/AudioOutput.cpp

```diff
@@ -616,12 +616,13 @@ bool AudioOutput::mix(void *outbuff, unsigned int frameCount) {
 					if (speech->bStereo) {
 						// Mix down stereo to mono. TODO: stereo record support
 						// frame: for a stereo stream, the [LR] pair inside ...[LR]LRLRLR.... is a frame
-						for (unsigned int i = 0; i < frameCount; ++i) {
-							recbuff[i] += (pfBuffer[2 * i] / 2.0f + pfBuffer[2 * i + 1] / 2.0f) * volumeAdjustment;
+						for (std::size_t i = 0; i < frameCount; ++i) {
+							recbuff.get()[i] +=
```

Comment:
Why the change?

Reply:
Because `recbuff` is of type `boost::shared_array` which overloads `operator[]` to take an argument of `std::ptrdiff_t` into which the compiler converts the `unsigned int` happily on x64 but produces a sign conversion warning on x86. And simply accessing the underlying buffer to use `operator[]` on that is shorter than an explicit `static_cast< std::ptrdiff_t >` :shrug:

Reply:
Makes sense.